src/components/carousel/carousel.test.js

```diff
@@ -0,0 +1,229 @@
+        </Box>
+      </Slide>
+      <Slide>
+        <Box
+          height={200}
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+          bg="#69418f"
+        >
+          <Typography variant="h1" color="#fff">
+            Slide 3
+          </Typography>
+        </Box>
+      </Slide>
+      <Slide>
+        <Box
+          height={200}
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+          bg="#69418f"
+        >
+          <Typography variant="h1" color="#fff">
+            Slide 4
+          </Typography>
+        </Box>
+      </Slide>
+      <Slide>
+        <Box
+          height={200}
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+          bg="#69418f"
+        >
+          <Typography variant="h1" color="#fff">
+            Slide 5
+          </Typography>
+        </Box>
+      </Slide>
+    </Carousel>
+  );
+};
+
+context("Testing Carousel component", () => {
+  describe("Should render Carousel component", () => {
+    it.each([
+      ["0", "Slide 1"],
+      ["1", "Full clickable slide"],
+      ["2", "Slide 3"],
+      ["3", "Slide 4"],
+      ["4", "Slide 5"],
+    ])("should verify slide %s title is %s", (index, title) => {
+      CypressMountWithProviders(<CarouselComponent />);
+
+      slide(index).should("have.text", title);
+    });
+
+    it.each([
+      ["1", "0", "Slide 1"],
+      ["2", "1", "Full clickable slide"],
+      ["3", "2", "Slide 3"],
+      ["4", "3", "Slide 4"],
+    ])(
+      "when left arrow is clicked to slide %s from slide %s title should be %s",
+      (index, indexSlide, title) => {
+        CypressMountWithProviders(<CarouselComponent slideIndex={index} />);
+
+        clickCarouselButton("left");
+
+        slide(indexSlide).should("have.text", title);
+      }
+    );
+
+    it.each([
+      ["0", "1", "Full clickable slide"],
+      ["1", "2", "Slide 3"],
+      ["2", "3", "Slide 4"],
+      ["3", "4", "Slide 5"],
+    ])(
+      "when right arrow is clicked from slide %s to slide %s title should be %s",
+      (index, indexSlide, title) => {
+        CypressMountWithProviders(<CarouselComponent />);
+
+        clickCarouselButton("right");
+
+        slide(indexSlide).should("have.text", title);
+      }
+    );
+
+    it("should verify left arrow button is disabled on first slide", () => {
+      CypressMountWithProviders(<CarouselComponent slideIndex={0} />);
+
+      previousArrowButton().should("be.disabled").and("have.attr", "disabled");
+    });
+
+    it("should verify right arrow button is disabled on last slide", () => {
+      CypressMountWithProviders(<CarouselComponent slideIndex={4} />);
+
+      nextArrowButton().should("be.disabled").and("have.attr", "disabled");
+    });
+
+    it.each([[true], [false]])(
+      "should verify slide Selector is enabled/disabled when 'enableSlideSelector' is set to %s",
```

Comment:
```suggestion
      "should verify when 'enableSlideSelector' is set to %s that slide Selector is  %s",
```

or change the position of values in array and leave the description as it is

Reply:
same here